backtrace-library/src/main/cpp/backtrace-native.cpp

```diff
@@ -321,4 +586,108 @@ Java_backtraceio_library_base_BacktraceBase_dumpWithoutCrash__Ljava_lang_String_
                                                                                    jboolean set_main_thread_as_faulting_thread) {
     DumpWithoutCrash(message, set_main_thread_as_faulting_thread);
 }
+
+bool InitializeLocalClientSideUnwinding(JNIEnv* env) {
+    // Initialize a shared memory region.
+    static const char *buffer = static_cast<char *>(buffer_create());
+
+    bun_buffer_init(&buf, (char*) buffer, BUFFER_SIZE);
+
+    bun_initialized.store(bun_handle_init(&handle, BUN_BACKEND_LIBUNWINDSTACK));
+
+    if (!bun_initialized) {
+        __android_log_print(ANDROID_LOG_ERROR, "Backtrace-Android", "Local client side unwinding initialization failed");
+        return false;
+    }
+
+    return true;
+}
+
+bool InitializeRemoteClientSideUnwinding(JNIEnv* env, jstring path) {
```

Comment:
I still don't know why we can't pass a boolean variable to the initialize method and invoke this method from the initialize method. It will completely simplify API in my opinion. Why do we want to have separated functions for crash reporting api? We can end up in a situation where user won't know what and how to do stuff because our api will be to complex to understand or simply it will be too complex to understand via readme.

Reply:
I think you're right, will make this change